prowler/providers/aws/services/emr/emr_cluster_publicly_accesible/emr_cluster_publicly_accesible.py

```diff
@@ -32,7 +32,10 @@ def execute(self):
                     master_node_sg_groups = deepcopy(
                         cluster.master.additional_security_groups_id
                     )
-                    master_node_sg_groups.append(cluster.master.security_group_id)
+                    if master_node_sg_groups:
```

Comment:
Do the same for the `slave_node_sg_groups` please.

Reply:
Done